File: jobs/2026-05-30__07-57-35/create_hello_world_go__PiD3Kd2/artifacts/user/helloworld-zr-pid3kd2/hello/hello.go
package hello

import (
	"context"
)

type WorldResponse struct {
	Message string
}

//encore:api public method=GET path=/hello/:name
func World(ctx context.Context, name string) (*WorldResponse, error) {
	return &WorldResponse{Message: "Hello, " + name + "!"}, nil
}

File: jobs/2026-05-30__07-57-35/migrate_express_ts__DJJg6q4/artifacts/user/helloworld/hello/hello.ts
import { api } from "encore.dev/api";

interface GreetParams {
  name: string;
}

interface GreetResponse {
  message: string;
}

export const greet = api(
  { method: "GET", path: "/greet/:name", expose: true },
  async ({ name }: GreetParams): Promise<GreetResponse> => {
    return { message: `Hello, ${name}!` };
  }
);
